packages/KDropdownMenu/KDropdownMenu.vue

```diff
@@ -0,0 +1,239 @@
+  >
+    <KToggle v-slot="{toggle, isToggled}">
+      <KPop
+        v-bind="boundKPopAttributes"
+        :on-popover-click="() => {
+          toggle();
+          return isToggled
+        }"
+        @opened="() => {
+          toggle()
+          $emit('toggleDropdown', true)
+        }"
+        @closed="() => {
+          toggle()
+          $emit('toggleDropdown', false)
+        }"
+      >
+        <div class="d-flex">
+          <component
+            :is="!!disabledTooltip ? 'Kooltip' : 'div'"
+            :label="disabledTooltip"
+            :position="!!disabledTooltip ? 'bottom' : undefined"
+            :position-fixed="!!disabledTooltip ? true : undefined"
+            :max-width="!!disabledTooltip ? '240' : undefined"
+            class="dropdown-trigger"
+          >
+            <slot
+              :is-open="isToggled"
+              name="default"
+            >
+              <!-- Must wrap in div to allow tooltip when disabled -->
+              <div>
+                <KButton
+                  v-if="label"
+                  :disabled="disabled"
+                  :is-open="showCaret || appearance === 'selectionMenu' ? isToggled : undefined"
+                  :class="{ 'is-active': showCaret ? isToggled : undefined }"
+                  :appearance="appearance === 'selectionMenu' ? 'outline' : 'primary'"
+                  class="k-dropdown-btn"
+                >
+                  {{ label }}
+                </KButton>
+              </div>
+            </slot>
+          </component>
+        </div>
+        <template #content>
+          <ul class="k-dropdown-list dropdown-list">
+            <slot
+              :items="items"
+              :is-open="isToggled"
+              name="items"
+            >
+              <KDropdownItem
+                v-for="(item, idx) in items"
+                v-bind="item"
+                :key="`${item.label.replace(' ', '-')}-${idx}`"
+                :item="item"
+                :selection-menu-child="appearance === 'selectionMenu'"
+              />
+            </slot>
+          </ul>
+        </template>
+      </KPop>
+    </KToggle>
+  </div>
+</template>
+
+<script>
+import KButton from '@kongponents/kbutton/KButton.vue'
+import Kooltip from '@kongponents/kooltip/KoolTip.vue'
+import KPop from '@kongponents/kpop/KPop.vue'
+import KToggle from '@kongponents/ktoggle/KToggle'
+import KDropdownItem from './KDropdownItem.vue'
+
+const defaultKPopAttributes = {
+  hideCaret: true,
+  popoverClasses: 'k-dropdown-popover mt-1',
+  popoverTimeout: 0,
+  positionFixed: true,
+  placement: 'bottomStart'
+}
+
+export default {
+  name: 'KDropdownMenu',
+  components: {
+    KButton,
+    KDropdownItem,
+    Kooltip,
+    KPop,
+    KToggle
+  },
+  props: {
+    appearance: {
+      type: String,
+      default: 'menu',
+      validator: (value) => {
+        return [
+          'menu',
+          'selectionMenu'
+        ].indexOf(value) !== -1
```

Comment:
Just to make it like our typical usage 
```suggestion
        ].includes(value)
```